Use python3-zstd instead of python3-pyzstd, as the latter is not available
in Ubuntu (prior to oracular).

--- a/src/calibre/test_build.py
+++ b/src/calibre/test_build.py
@@ -104,10 +104,10 @@ class BuildTest(unittest.TestCase):
         lzma.open
 
     def test_zstd(self):
-        from pyzstd import compress, decompress
+        from zstd import ZSTD_compress, ZSTD_uncompress
         data = os.urandom(4096)
-        cdata = compress(data)
-        self.assertEqual(data, decompress(cdata))
+        cdata = ZSTD_compress(data)
+        self.assertEqual(data, ZSTD_uncompress(cdata))
 
     def test_html5lib(self):
         import html5lib.html5parser  # noqa: F401
